Nos/Views/ScrollViewIfNeeded.swift

```diff
@@ -0,0 +1,35 @@
+import SwiftUI
+
+/// A view that places its contents in a ScrollView if they exceed the available space.
+/// This is useful because some views won't expand to fill the available
+/// space when they are put in a ScrollView.
+struct ScrollViewIfNeeded<Content>: View where Content: View {
+    @ViewBuilder var content: Content
+    
+    var axes: Axis.Set = [.vertical]
+    
+    var body: some View {
+        ViewThatFits(in: axes) {
```

Comment:
🤯 I can't believe I didn't know about `ViewThatFits` until just now.

Reply:
It was an iOS 16 introduction but yeah, it's amazing.